### PR TITLE
Simple stripe implementation

### PR DESCRIPTION
**Pull request by Sc0rpie**
What's been added/updated

A simple card payment via /payment/process-card has been added that takes StripeSecretKey from the user's assigned business. For now it does not create a Payment object (does not store the data in db) because Orders, Giftcards and customers were missing.

A simple cash payment via /payment/process-cash has been added that just takes the data (for now only cash) and returns a unique payment id (also not being saved/stored in DB). Currently no Auth rules for this one while not final

Business has been updated to have StripeSecretKey and StripePublishKey instead of StripeAccId

### CODE REVIEW

**Review by MrMnt — DISMISSED** [review]
Nice! Not sure how everything works / will work, but approving this initial implementation

File: OmgvaPOS/src/PaymentManagement/Controllers/PaymentController.cs [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+        }
+
+        [HttpPost]
+        [Route("process-card")]
+        public IActionResult ProcessCardPayment([FromBody] PaymentRequest request)
+        {
+            var businessId = JwtTokenHandler.GetTokenBusinessId(HttpContext.Request.Headers.Authorization);
+            if (businessId == null)
+            {
+                return Unauthorized(new { error = "Unauthorized" });
```
Comment (AugustinasJarockis) [inline OmgvaPOS/src/PaymentManagement/Controllers/PaymentController.cs:29]: This should be Forbid. But maybe Mantas will fix it. Feel free to resolve, but keep in mind that we use forbid when authenticated user lacks authorisation.

File: OmgvaPOS/src/PaymentManagement/Controllers/PaymentController.cs [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        }
+        
+        [HttpPost]
+        [Route("process-cash")]
+        public IActionResult ProcessCashPayment([FromBody] PaymentRequest request)
+        {
+            var businessId = JwtTokenHandler.GetTokenBusinessId(HttpContext.Request.Headers.Authorization);
+            if (businessId == null)
+            {
+                return Unauthorized(new { error = "Unauthorized" });
```
Comment (AugustinasJarockis) [inline OmgvaPOS/src/PaymentManagement/Controllers/PaymentController.cs:100]: Same here

**Review by AugustinasJarockis — APPROVED** [review]

**Review by AugustinasJarockis — APPROVED** [review]
Looks good.

**Pull request merged**